docs/Pace_Sbom_Vex_Flags_Prioritization/README.md

```diff
@@ -0,0 +1,376 @@
+(e.g. the medical devices themselves).
+The systems used to execute the PACE functionality,
+or other cybersecurity functionality that takes action
+based on PACE evaluation,
+will be called "infrastructure".
+The systems that exist to carry out the mission of the organization
+(eg medical devices in a hospital) will be called assets
+which will be described in a later section.
+
+### IACD
+Execution of the use cases
+will follow the
+[IACD model](https://www.iacdautomate.org/)
+of sensing, sense-making, decision-making, acting.
+
+Sensing will be gaining input such as asset inventory,
+SBOM, VEX, situational awareness, etc.
+Sense-making will be combining inputs and augmenting information
+to provide to decision making.
+E.g. the combining of SBOM, VEX, and NVD information.
+Decision making will be the control flow via predetermined playbooks
+(some without human involvement, and some with humans in the decision path).
+Acting will be the command and control
+sent to the various security technologies
+to implement the decision.
+
+The use cases will, to the extent possible, attempt to be
+vendor-agnostic and describe the functionality, not a
+particular product or systems.
+The exceptions may be PACE or other OCA subprojects (eg Kestrel, Stixshifter) where the OCA open source project may be mentioned.
+
+## Assets
+These are the systems being protected.
+The systems doing the protecting will be called
+infrastructure as described previously.
+
+## Attributes
+For the purpose of these use cases,
+the primary attributes involve software
+composition e.g. SBOM, VEX, NVD, etc.
+Each of these may have individual parameters of their own,
+which will be dealt with in the next section.
+Other attributes which will affect the security posture
+will be added later such as:
+- threat intel (STIX) from a relevant ISAC,
+- Kestrel info on an intruder already in the system,
+- OSCAL
+- SCAP
+- ACAS
+- XCCDF
+- ...
+
+## Parameters
+### SBOM
+- SBOM generation
+   + source code SBOM
+   + build time SBOM
+   + post-build SBOM
+- Completeness
+   + complete
+   + contains known-unknowns (i.e. specific denoted known-unknowns)
+   + incomplete (i.e. unknowns are not specifically delineated)
+   + not_present (i.e. there will be use cases for when there is no SBOM for that device)
+- format/serialization/version
+   + format = CycloneDX
+      - serialization = xml or json (distinction will not matter for these use cases at this time)
+   + format = SPDX
+      - serialization = xml, json, ... (distinction will not matter for these use cases at this time)
+   + For the purpose of these use cases at the moment, version of the format or serialization is being ignored  - but it is a parameter to be used by PACE
+- version
+   + for the purpose of these use cases at the moment, version of the SBOM is being ignored - but it is a parameter to be used by PACE
+- note that for a given device, PACE PAR may contain more than one SBOM, of more than one format.
```

Comment:
Don't know what "PAR" is in "PACE PAR". Please expand.

Reply:
@dlemire60 - I'm not sure the use cases should be tutorial. Maybe they should refer to other docs to understand SBOMs or VEXs or PACE. I am ok with expanding acronyms. I got Posture Collection System, Posture Evaluation System and Posture Attribute Repository from other PACE docs. If I should be using different, let me know.